source/install/troubleshooting.rst

```diff
@@ -629,3 +629,293 @@ To help us narrow down whether it’s a server configuration issue, device speci
+
+.. code-block:: none
+    
+   SELECT Value FROM Configurations WHERE Active = 1;
+
+**Reverse Proxy configuration**
+
+Nginx usually splits its configuration into two parts: the main server configuration at ``/etc/nginx/nginx.conf``, and a virtual server configuration. On Ubuntu, this is stored in ``/etc/nginx/sites-available``. Providing both of these configuration files is helpful, but providing the latter is more important.
+
+**SAML configuration**
+
+If the issue you're seeing is with SAML login, we will need to see the full configuration for the Mattermost service in the SAML provider. Providing screenshots similar to the ones in the setup documentation is sufficient because most SAML providers are configured using a web interface.
+
+**LDAP configuration**
+
+The LDAP administrator should confirm the correct values for the following Mattermost LDAP settings:
+
+- LDAP server hostname
+- LDAP connection port, security, and certificates
+- BaseDN, bind username, and bind password
+- User, Group, Guest, and Admin filters
+- Display attributes
+
+These can be provided as a text file or as screenshots from the LDAP server.
+
+**Other configurations**
+
+If you're experiencing an issue on mobile, and you're using an MDM or VPN to connect to the server, those configurations will be necessary to diagnose the problem. A System Administrator for the external system should be able to provide you with the configuration.
+
+Logs
+~~~~
+
+Why we need it
+^^^^^^^^^^^^^^
+
+Nearly all computer systems have logs of errors and application behavior that can show us what's happening when an application is running. Error logs are invaluable when diagnosing a problem, but only if they're as complete as possible.
+
+What logs are available
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Mattermost**
+
+Mattermost has two log files, one for general messages and the other for notification-related messages. These are found at:
+
+* ``/opt/mattermost/logs/mattermost.log``
+* ``/opt/mattermost/logs/notification.log``
+
+**Proxy**
+
+The location of these depend on your proxy configuration, but a good place to start looking is in ``/var/log``. Your proxy administrator should be able to help you find the logs.
+
+**Database**
+
+MySQL and PostgreSQL have different logs, and their location varies based on your configuration. If the issue is related to database connectivity, check the database documentation to locate the logs.
+
+**SAML, LDAP, and other systems**
+
+Your organization's System Administrator should be able to find these for you.
+
+How to access logs
+^^^^^^^^^^^^^^^^^^
+
+**Mattermost**
+
+Make sure debug logging is enabled so that we can get the most information from the logs. To do this, edit your Mattermost configuration to set the File Log Level to **DEBUG**.
```

Comment:
```suggestion
Make sure debug logging is enabled so that we can get the most information from the logs. To do this, go to **System Console > Environment > Logging** and then set console and file log level to **DEBUG**.
```